docs/_includes/Example-Old.dfy

```diff
@@ -0,0 +1,28 @@
+class A {
+
+  var value: int;
+
+  method m(i: int)
+    requires i == 6
+    requires value == 42
+    modifies this
+  {
+    var j: int := 17;
+    value := 43;
+    label L: {}
```

Comment:
This `label` statement is used to label the empty block `{}`. That's okay. But it could just as well label the statement on the next line. So, I suggest removing the `{}` from L12 and from L15.

The same comment applies to the examples above.

Reply:
OK